test/unit/update_versions_file_test.rb

```diff
@@ -44,7 +44,7 @@ class UpdateVersionsFileTest < ActiveSupport::TestCase
       end
 
       should "include platform release" do
-        expected_output = "rubyrubyruby 0.0.1-jruby,0.0.1 qw212r\n"
+        expected_output = "rubyrubyruby 0.0.1,0.0.1-jruby qw212r\n"
```

Comment:
> There's also this "ordering change" that I'm not sure where it comes from man_shrugging

>  The order doesn't matter at the moment, 

I was wrong about this as well. The order does matter as the checksum of these two lines is different.
```
2.6.5 :014 > Digest::MD5.hexdigest("rubyrubyruby 0.0.1-jruby,0.0.1 qw212r\n")
 => "b3eb3b29e9b520cb64bb3d4038e06bbb" 
2.6.5 :015 > Digest::MD5.hexdigest("rubyrubyruby 0.0.1,0.0.1-jruby qw212r\n")
 => "c3892a889a4d9c7a2d2672f63d7e9a95" 
```